### ISSUE TITLE
Getting issue after creating cookie through cookie manager

### ISSUE THREAD

**Issue by vjsampas** [eigengrau-ch/strapi-plugin-cookie-manager#16]
Steps to reproduce:

1. Install plugin as per steps mentioned
2. Launch strapi admin
3. Go to cookie manger plugin
4. Create category - created without any issue
5. Create cookie inside category (Just with mandatory field) - Blank screen with error in developer tool (Please see image attached)

Am I missing anything or something wrong with plugin?

Environment details:
Strapi version:  4.7.0
codemirror: 5.65.11
Cookie-manager version : 1.0.8

![image](https://user-images.githubusercontent.com/128164448/225942922-41f66bf0-509c-40f4-b1db-8aff47d2eb11.png)

**Comment by eigengrau-ch** [comment]
Hi @vjsampas, thank you for reaching out.

I could absolutely be that the newer strapi versions are breaking the plugin as of now. Unfortunately I was very busy with client projects the past months which delayed our version upgrades and with it possible error fixes.

In the coming weeks I will update everything to the newest versions which probably will fix your error too.

**Comment by eigengrau-ch** [comment]
Hi @vjsampas, thank you for waiting.

In the meanwhile I have upgraded my projects to the newest strapi version without any issues regarding this plugin. Except the package blocking node 18 upgrade, which has been solved recently.

Never than less, I booted up a fresh new strapi, installed the plugin and started filling in content which was mandatory exactly as you described. Unfortunately I could not reproduce your error and the message kind of leaves me pretty clueless on where it could might have occurred. I guess easiest for now would be, that you upgrade your strapi to the latest version. Let me know if this didn't fix your error.
